diagnostic.md

```diff
@@ -35,7 +36,10 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+      The first route is nested, so when the user navigates to /products/3,
+      product 3 will be rendered within the products template, whereas with the
+      second route, if the user goes to the same URL, the same product will be
+      render but within the application template.
```

Comment:
You are correct.  One of the biggest distinctions is where Ember will render the content.  The nested `'products'/'product'` instructs Ember to render `app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.

What else is different?  These two route definitions are identical  URL paths.  Nested route definitions, do require a nested file structure.